### DISCUSSION TITLE
bigfix/10.0/lifecycle/Lifecycle/ServerAutomation/SAUsersGuide/Server_Automation_virtualization/sa_c_vmware

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#149] [Q&A]
# bigfix/10.0/lifecycle/Lifecycle/ServerAutomation/SAUsersGuide/Server_Automation_virtualization/sa_c_vmware

You can deploy and configure a management extender plug-in for VMware. You can then manage VMware virtual machines from within BigFix the console.

https://help.hcl-software.com/bigfix/10.0/lifecycle/Lifecycle/ServerAutomation/SAUsersGuide/Server_Automation_virtualization/sa_c_vmware.html

**Comment by MrKv2000** [comment]
" You you can power off" in this sentence two "you" words are encountered.
